pkg/apis/devices/v1beta1/device_instance_types.go

```diff
@@ -230,6 +232,19 @@ type TDEngineClientConfig struct {
 	DBName string `json:"dbName,omitempty"`
 }
 
+type DBMethodMySQL struct {
+	MySQLClientConfig *MySQLClientConfig `json:"mySQLClientConfig,omitempty"`
```

Comment:
```suggestion
	MySQLClientConfig *MySQLClientConfig `json:"mysqlClientConfig,omitempty"`
```

Reply:
Because L164 is mysql, use mysqlClientConfig here may be better?

Reply:
ok